parquet-hadoop/README.md

```diff
@@ -324,9 +324,20 @@ ParquetInputFormat to materialize records. It should be a the descendant class o
 **Property:** `parquet.read.schema`  
 **Description:** The read projection schema.
 
-
 ## Class: UnmaterializableRecordCounter
 
 **Property:** `parquet.read.bad.record.threshold`  
 **Description:** The percentage of bad records to tolerate.  
 **Default value:** `0`
+
+## Class: ZstandardCodec
+
+**Property:** `parquet.compression.codec.zstd.level`
+**Description:** The compression level of ZSTD. The valid range is 1~22. Generally the higher compression level, the higher compression ratio can be achieved, but the writing time will be longer.     
```

Comment:
In markdown you have to do some extra if you want to force new lines. With a simple linebreak the two lines will be written together. There are some tricks [here](https://gist.github.com/shaunlebron/746476e6e7a4d698b373) for enforcing line breaks.
(You may check the rendered format on [github](https://github.com/shangxinli/parquet-mr/tree/master_new/parquet-hadoop)).

Reply:
Thanks for pointing out! I think it is OK to keep them in same line unless you have strong opinion we should have a separate line. I looked at the existing lines above and I see some of them are longer than mines.

Reply:
I was trying to say that _Property_, _Description_ and _Default value_ should be separate paragraphs. Currently they are not, the _Description_ is rendered just after _Property_ in the same line even if in the markdown they are separated. You should use one of the techniques described under the link to force the new line there.

Reply:
I see. I added double space in the end of Property, Description sentences. I checked it in Intellij and it works. I also see the above sections used the trailing double space.